### PR TITLE
Mantine Loader

### LINKED ISSUE #32309: Loader component
Docs

- [Product doc](https://www.notion.so/metabase/Design-system-Loader-component-1038a6a0661b471ea7710407b852faf2?pvs=4)
- [Figma component](https://www.figma.com/file/NUXRUa9Ot3HvgC1WwIA4UH/Loader?type=design&node-id=418-2983&mode=design&t=aPGdEhPIoQXkWpfC-11)

```[tasklist]
### Tasks
- [ ] Export the [Mantine](https://mantine.dev/core/button/) `Button` component from the `metabase/frontend/src/ui` according to the component’s category.
- [ ] Make sure the component style matches the component in Figma. You can add component specific styling to the `components:` section of `frontend/src/metabase/ui/theme.ts`
- [ ] Create a storybook file for the new component in the same directory. You can use `frontend/src/metabase/ui/components/inputs/Radio/Radio.stories.mdx` as an example.
- [ ] Add the following caveats to the Loader Story

```

Caveats
- Only use the “OVAL” variant unless otherwise called out
- Prefer usage of loader sizes less than XL unless specifically called out

### PR DESCRIPTION
**Pull request by ranquild**
Closes https://github.com/metabase/metabase/issues/32309

Adds a wrapper for `Loader` component. 

The component doesn't use `useStyles` API so it's not possible to apply style overrides at all. That's why to override sizes I had to create out own wrapper component. 

How to verify:
- `yarn storybook`
- Open http://localhost:6006/?path=/docs/feedback-loader--sizes

Screenshot 2023-08-18 at 14 31 33

### CODE REVIEW

**Review by oisincoveney — APPROVED** [review]
Tested locally, looks good! 😄

**Review by dhuniverse — APPROVED** [review]
It all looks good to me.

File: frontend/src/metabase/ui/components/feedback/Loader/Loader.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const SIZES: Record<string, string> = {
+  xs: "1rem",
+  sm: "1.25rem",
+  md: "1.5rem",
+  lg: "2rem",
+  xl: "3.5rem",
+};
+
+export const Loader = ({ size = "md", ...props }: LoaderProps) => (
+  <MantineLoader {...props} size={SIZES[size] ?? size} />
```
Comment (uladzimirdev) [inline frontend/src/metabase/ui/components/feedback/Loader/Loader.tsx:13]: I'm surprised their types allow to pass any string as size, but should we?

Comment (ranquild) [reply frontend/src/metabase/ui/components/feedback/Loader/Loader.tsx:13]: Currently all mantine components do. I'm not sure now, but we should keep an eye of this causes issues

**Pull request merged**